src/sinks/util/retries.rs

```diff
@@ -193,12 +193,14 @@ where
                     );
                     Some(self.build_retry())
                 } else {
+                    // For now, retry request in all cases
                     error!(
-                        message = "Unexpected error type; dropping the request.",
+                        // message = "Unexpected error type; dropping the request.",
+                        message = "Unexpected error type encountered.",
```

Comment:
append `..retrying` in message just to be clear for someone who just see this log.